localization/replace/english/culture/traditions/wc_cultural_traditions_l_english.yml

```
@@ -0,0 +1,85 @@
+ tradition_himalayan_settlers_desc:0 "Grummles have lived around Mount Neverest and navigated the Burlap Trail for as long as lorewalkers keep records. In times of strife, the winding wends and craggy peaks of Kun-Lai make good shelter and even better outposts for guerilla attacks that melt away into the wind and snow."
+ 
+ tradition_mountaineer_ruralism_name:0 "Hill Lodges"
+ tradition_mountaineer_ruralism_desc:0 "From the rugged highlands to the verdant valleys below, the trappers of the Grizzly Hills track across the varied geography with practiced ease and have established settlements in areas otherwise thought unsuitable to habitation."
+ 
+ tradition_nubian_warrior_queens_name:0 "Warrior Queens"
+ tradition_nubian_warrior_queens_desc:0 "This culture is built around a social hierarchy dominated by female leadership and there is no better embodiment of their virtue than taking the field of battle."
+
+ tradition_desert_nomads_name:0 "Desert Travelers"
+ tradition_desert_nomads_desc:0 "This culture is no stranger to the harsh environment of the desert and has mastered the use of camels to aid them in travel, warfare, and everyday life."
+ 
+ tradition_caravaneers_name:0 "Caravaneers"
+ tradition_caravaneers_desc:0 "This culture maintains crucial merchant networks far and wide across its region, and its people are strongly associated with the caravan trade."
+ 
+ tradition_horse_lords_name:0 "Masters of the Plains"
+ tradition_horse_lords_desc:0 "Whether upon the saddle or by one's own hooves, warriors of this culture specialise in striking foes with lightning speed and exploiting vulnerabilities in enemy formations to devastating effect."
+ 
+ tradition_fp1_coastal_warriors_name:0 "Trials of Valour"
+ tradition_fp1_coastal_warriors_desc:0 "This culture holds no honor greater than dying in glorious battle. Berserkers harness this scorn for mortality and turn themselves into unstoppable juggernauts of death through practice and ritual."
+ 
+ tradition_fp1_northern_stories_name:0 "Tales of Legend"
+ tradition_fp1_northern_stories_desc:0 "Life is lived, told, and remembered through the tapestry of our people's history. We honor those who came before by standing tall and speaking true, knowing one day we will join the ranks of fabled lore."
+ 
+ tradition_hird_name:0 "Trials of Valour"
+ tradition_hird_desc:0 "This culture holds no honor greater than dying in glorious battle. Berserkers harness this scorn for mortality and turn themselves into unstoppable juggernauts of death through practice and ritual."
+ 
+ tradition_land_of_the_bow_name:0 "Elven Marksmanship"
+ tradition_land_of_the_bow_desc:0 "This culture is fabled for its mystical mastery of the bow, perfected with deadly grace through millennia of accumulated experience."
+ 
+ tradition_forest_wardens_desc:0 "This culture has made its home in the uncharted depths of ancient forests and fiercely guards the pristine sanctity of their ancestral homes from unwelcome trespassers."
```

Same suggestion as up above in the cultural innovation file:
```suggestion
 tradition_forest_wardens_desc:0 "Living in the uncharted depths of ancient forests, the pristine sanctity of our ancestral homes is fundamental to our wardens. Any unwelcome trespasser might realise that before setting eyes on them."
```